servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -35,39 +35,57 @@
 /**
  * Task that carries out the necessary block worker to master communications, including register and
  * heartbeat. This class manages its own {@link tachyon.client.WorkerBlockMasterClient}.
- *
+ * <p/>
  * When running, this task first requests a block report from the
  * {@link tachyon.worker.block.BlockDataManager}, then sends it to the master. The master may
  * respond to the heartbeat with a command which will be executed. After which, the task will wait
  * for the elapsed time since its last heartbeat has reached the heartbeat interval. Then the cycle
  * will continue.
- *
+ * <p/>
  * If the task fails to heartbeat to the master, it will destroy its old master client and recreate
  * it before retrying.
  */
 public final class BlockMasterSync implements Runnable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private static final int DEFAULT_BLOCK_REMOVER_POOL_SIZE = 10;
-  /** Block data manager responsible for interacting with Tachyon and UFS storage */
+  /**
+   * Block data manager responsible for interacting with Tachyon and UFS storage
+   */
```

Comment:
For all of these variables, if the javadoc fits on one line, we should just keep the original formatting.